src/test/java/com/wikia/webdriver/testcases/logintests/ForgottenPasswordTests.java

```diff
@@ -49,13 +52,39 @@ public void remindPasswordFromLoginDropdown() {
     dropdown.verifyUserLoggedIn(userName);
   }
 
+  @Test()
+  @Execute(onWikia = "agas")
```

Comment:
this environment should be standardized because if this wiki is not one dedicated for automated tests only (eg. manual testing is performed on that wiki as well), a different dedicated silent wiki should be used.